src/DotNetLightning.Core/Utils/NBitcoinExtensions.fs

```diff
@@ -35,6 +36,9 @@ module NBitcoinExtensions =
     type Key with
         static member BytesLength: int = 32
 
+        static member FromHash(preimage: array<byte>): Key =
```

Comment:
The name is confusing, it sounds like the argument is hash itself.
I suggest `FromHashOf` instead.

Reply:
Fixed.